README.md

```diff
@@ -1,6 +1,53 @@
-<h1 align="center">Draw-it-out: Online Whiteboard App</h1>
-
+<img  src="https://readme-typing-svg.herokuapp.com?size=40&width=900&height=80&lines=Welcome-to-Draw-it-out"/>
```

Comment:
### Hi there @ananyag309 👋🏻 

You need to wrap the Readme Typing SVG into the `div` which acts as parent for Draw-it-Out Logo. Because Currently the Typing SVG is not centered horizontally.

```suggestion
<div align="center">
  <img src="https://readme-typing-svg.herokuapp.com?size=40&width=900&height=80&lines=Welcome-to-Draw-it-out"/>
  <img src="./src/assets/images/logo.png" width="200px" height="200px"/>
</div>
```